Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -1270,42 +1271,44 @@ WebIDL::ExceptionOr<JS::NonnullGCPtr<Event>> Document::create_event(DeprecatedSt
 
     // 2. If interface is an ASCII case-insensitive match for any of the strings in the first column in the following table,
     //      then set constructor to the interface in the second column on the same row as the matching string:
-    auto interface_lowercase = interface.to_lowercase();
```

Comment:
This one is unfortunate: We lose the caching of the lowercase string. Then again, we don't have to allocate a new String, so you win some and you lose some.